### DISCUSSION TITLE
Item Stats: Only show tooltips when the equipment stats interface is open

### DISCUSSION THREAD

**Discussion by ghost** [runelite/runelite#18179] [Ideas]
I would really love a tickbox option to only have the item stats tooltips appear when I have the equipment stats interface up. That way we would be able to see the useful equipment info from item stats (as well as consumable info and weight), but without the big tooltips filling up the inventory when you mouse over it normally. 

If this is added, I guess if 'show stats in bank' is also enabled it would mean you can see the stats both in the bank and the equipment stats interface.

**Comment by ColdDudeX** [comment]
Maybe an option: 'Show Stats in Inventory,' so we can disable it only when using the inventory. Keeping both bank and equipment interface.
